src/components/Cards/CardContainer.js

```diff
@@ -0,0 +1,23 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Equalizer from 'react-equalizer';
+
+const CardContainer = ({ nodes, children, className, ...props }) => {
+  const classes = ClassNames('container-fluid container-cards-pf', className);
+
+  return (
```

Comment:
i think we can expose a boolean `equalizer` prop and then render something like:
```
  if(equalizer){
  return (
    <Equalizer nodes={nodes} className={classes} {...props}>
      {children}
    </Equalizer>
  ); } else {
    <div className={classes} {...props}>
      {children}
    </div>
}
```